packages/map-template/src/components/Map/Map.jsx

```diff
@@ -208,11 +208,11 @@ function Map({ onLocationClick, onVenueChangedOnMap, useMapProviderModule, onMap
         });
 
         // Detect when the mouse hovers over a location and store the hovered location
-        // If the location is non-selectable, remove the hovering by calling the unhoverLocation() method.
+        // If the location is non-selectable/inherited from type, remove the hovering by calling the unhoverLocation() method.
```

Comment:
This can be read as the location is inherited from the type. How about something like

```suggestion
        // If the location is non-selectable (either set on the Location or inherited from the type), remove the hovering by calling the unhoverLocation() method.
```